README.md

```diff
@@ -59,7 +59,7 @@ based applications requiring SSH support.
 
 * **Authentication methods**: hostbased, publickey, [OpenSSH host-based public-key](https://github.com/openssh/openssh-portable/blob/1781f507c11/PROTOCOL#L349), keyboard-interactive, password
 * **Ciphers**: aes128cbc, aes128ctr, aes192cbc, aes192ctr, aes256cbc, aes256ctr, arcfour128, arcfour256, blowfishcbc, tripledescbc,
-aes128-gcm@openssh.com, aes256-gcm@openssh.com, chacha20-poly1305@openssh.com
+aes128-gcm@openssh.com, aes256-gcm@openssh.com, chacha20-poly1305@openssh.com, 3des-cbc
```

Comment:
I see "tripledescbc" in the line above. That's 3des-cbc. And "blowfishcbc" should be "blowfish-cbc".

When that was written, the Java enum constant names were used by mistake instead of the algorithm names.

Reply:
Done.